src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -171,6 +171,11 @@ public static class RSessionEvaluationCommands {
             return evaluation.EvaluateAsync<JArray>(script, REvaluationKind.Normal);
         }
 
+        public static Task<JArray> UserLibraryPathsAsync(this IRExpressionEvaluator evaluation) {
+            var script = @"rtvs:::packages.userlibpaths()";
+            return evaluation.EvaluateAsync<JArray>(script, REvaluationKind.Normal);
```

Comment:
... which the code at the call site assumes to be the case, since it uses a cast that will fail if something is not actually a string.